plugins/utilities.go

```diff
@@ -25,13 +25,21 @@ func PointWithTag(key string, value interface{}, tagKey string, tagValue string)
 }
 
 func PointWithTags(key string, value interface{}, tags map[string]string) *timeseries.Point {
-       return timeseries.NewPoint(
-               key,
-               tags,
-               map[string]interface{}{
-                       "value": value,
-               },
-       )
+	return timeseries.NewPoint(
+		key,
+		tags,
+		map[string]interface{}{
+			"value": value,
+		},
+	)
+}
+
+func PointsWithTags(key string, values map[string]interface{}, tags map[string]string) *timeseries.Point {
```

Comment:
Can we find a better naming for this? I this it's confusing to name it `Points...` when it returns a single `Point`.

Reply:
@abrander PointValuesWithTags()?